<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoriasAdd</class>
 <widget class="QDialog" name="CategoriasAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutCabecera"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutBotones">
    <item>
     <widget class="QPushButton" name="btnRestablecer">
      <property name="text">
       <string>Restablecer</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGuardar">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>204</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layoutPrincipal">
    <item>
     <widget class="QLabel" name="textoCategoria">
      <property name="text">
       <string>Introduce el nombre de la categoria (máx. 20 caracteres):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cajaCategoria"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="textoExtensiones">
      <property name="text">
       <string>Introduce las extensiones de la categoría (mín. 1 extensión)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="cajaExtensiones">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mensajeError">
      <property name="text">
       <string>Aqui va el mensaje de error.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
